Source/MSBuildTools.Unity.NuGet/MSBuildForUnity.csproj

```diff
@@ -27,13 +27,20 @@
     <!-- Major is hard-coded and should be incremented on breaking changes -->
     <MajorVersion>0</MajorVersion>
 
-    <MinorVersion>1</MinorVersion>
+    <!-- Minor version is hard-coded and should be incremented on feature additions that aren't breaking changes.-->
+    <MinorVersion>8</MinorVersion>
 
-    <!-- Revision is defaulted to 0, but if running in the lab where BuildID is specified then default the revision to that value. -->
+    <!-- Revision (semver Patch) version is hard-coded and should be incremented when a bug fix is made.-->
     <RevisionVersion>0</RevisionVersion>
-    <RevisionVersion Condition="'$(BUILD_BUILDID)' != ''">$(BUILD_BUILDID)</RevisionVersion>
-
-    <PackageVersion>$(MajorVersion).$(MinorVersion).$(RevisionVersion)</PackageVersion>
+    
+    <Version>$(MajorVersion).$(MinorVersion).$(RevisionVersion)</Version>
+    <!-- Version is based on Major.Minor.Revision as defined above, however, in a lab BUILD_BUILDID will be set so we pull in that as the pre-release version. -->
+    <Version Condition="'$(BUILD_BUILDID)' != ''">$(Version)-$(BUILD_BUILDID)</Version>
```

Comment:
If you are switching the NuGet package version to use the pre-release suffix, then you should remove the NuGet package badge from the top level readme.md as ADO does not provide package badges for pre-release builds.

Reply:
Could I just update it to show latest stable when I go ahead and release 0.8.0?